provider/src/main/java/org/sunbird/aws/snsclient/SnsNotificationService.java

```diff
@@ -0,0 +1,26 @@
+package org.sunbird.aws.snsclient;
+
+import com.amazonaws.services.sns.model.MessageAttributeValue;
+import com.amazonaws.services.sns.model.PublishRequest;
+import com.amazonaws.services.sns.model.PublishResult;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by nickpack on 09/08/2017.
+ */
+public class SnsNotificationService {
```

Comment:
@shriharshs This class should also implement a common interface for SMS sending channels.